src/ray/core_worker/core_worker.cc

```diff
@@ -1055,6 +1055,9 @@ Status CoreWorker::CreateOwnedAndIncrementLocalRef(
                             });
     // Block until the remote call `AssignObjectOwner` returns.
     status = status_promise.get_future().get();
+    // Must call `AddNestedObjectIds` after finished assign owner
```

Comment:
Can you provide some context to this comment and add a link here to the github issue?

Reply:
@stephanie-wang sure.